src/main/java/picard/sam/markduplicates/ElcHashBasedDuplicatesFinder.java

```diff
@@ -40,13 +40,20 @@
  */
 class ElcHashBasedDuplicatesFinder extends ElcDuplicatesFinder {
 
-    // We split each read on several parts and each part has length hashLength.
-    // hashLength = (min read length) / (number of errors + 1)
-    private int hashLength = Integer.MAX_VALUE;
+    // We split each read on several parts, each part represent set of nucleotides,
+    // which behind each other on numberOfHashesInGroup.
+    // f. i.:
+    // 1 2 3 4 1 2 3 4 1 2 3 4
+    // A C A T T A C G G A T T
+    private int numberOfHashesInGroup = -1; //initial value
```

Comment:
rather than "initial value" please comment about the purpose of this variable. in this case I think it is the number of different groups the bases are split into"